//
// File: FSM_variable.cpp
//
// Code generated for Simulink model 'decision_simple'.
//
// Model version                  : 1.770
// Simulink Coder version         : 8.14 (R2018a) 06-Feb-2018
// C/C++ source code generated on : Tue May 14 19:44:10 2019
//
// Target selection: ert.tlc
// Embedded hardware selection: Freescale->S12x
// Code generation objectives: Unspecified
// Validation result: Not run
//
#include "include/decision/rtwtypes.h"
#include "include/decision/decision_simple_types.h"

// Exported data definition

// Definition for custom storage class: Global
uint8_T C1;
uint8_T E1;
uint8_T E2;
uint8_T E3;
uint8_T intersection_C1;
uint8_T intersection_state;
uint8_T lane_driving_C1;
uint8_T lane_driving_C2;
uint8_T lane_driving_C3;
uint8_T lane_driving_C4;
uint8_T lane_driving_E1;
uint8_T lane_driving_E2;
uint8_T lane_driving_F;
uint8_T lane_driving_state;
uint8_T upper_state;

//
// File trailer for generated code.
//
// [EOF]
//
